modules/core/src/webgl/texture.js

```diff
@@ -315,14 +315,14 @@ export default class Texture extends Resource {
     target = this.target,
     pixels = null,
     data = null,
-    width,
-    height,
+    width = this.width,
+    height = this.height,
     level = 0,
-    format = GL.RGBA,
-    type,
-    dataFormat,
+    format = this.format,
+    type = this.type,
+    dataFormat = this.dataFormat,
```

Comment:
Shall we align the naming with the [API spec](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/texImage2D) to prevent from mis-assignment?

`format` => `internalformat`
`dataFormat` => `format`

And update the jsDocs accordingly.

Reply:
That will be a breaking change. I will open an issue for v7.

Reply:
https://github.com/uber/luma.gl/issues/788